python/lsst/sdm_schemas/schemas/imsim.yaml

```diff
@@ -6015,7 +6015,608 @@ tables:
+    datatype: double
+    description: Ellipse rho (correlation coefficient) of the first Gaussian component in the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_gauss1_fluxfrac
+    "@id": "#Object.z_psfModel_TwoGaussian_gauss1_fluxfrac"
+    datatype: double
+    description: Fraction of the total flux (normalized to unity) of the first Gaussian component in the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_gauss2_sigma_x
+    "@id": "#Object.z_psfModel_TwoGaussian_gauss2_sigma_x"
+    datatype: double
+    description: Standard deviation of the second Gaussian component (x-axis) in the two-Gaussian PSF model (z-band).
+    fits:tunit: pixel
+  - name: z_psfModel_TwoGaussian_gauss2_sigma_y
+    "@id": "#Object.z_psfModel_TwoGaussian_gauss2_sigma_y"
+    datatype: double
+    description: Standard deviation of the second Gaussian component (y-axis) in the two-Gaussian PSF model (z-band).
+    fits:tunit: pixel
+  - name: z_psfModel_TwoGaussian_gauss2_rho
+    "@id": "#Object.z_psfModel_TwoGaussian_gauss2_rho"
+    datatype: double
+    description: Ellipse rho (correlation coefficient) of the second Gaussian component in the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_cen_x
+    "@id": "#Object.z_psfModel_TwoGaussian_cen_x"
+    datatype: double
+    description: Centroid (x-axis) of the two-Gaussian PSF model (z-band).
+    fits:tunit: pixel
+  - name: z_psfModel_TwoGaussian_cen_y
+    "@id": "#Object.z_psfModel_TwoGaussian_cen_y"
+    datatype: double
+    description: Centroid (y-axis) of the two-Gaussian PSF model (z-band).
+    fits:tunit: pixel
+  - name: z_psfModel_TwoGaussian_n_iter
+    "@id": "#Object.z_psfModel_TwoGaussian_n_iter"
+    datatype: int
+    description: Number of iterations in the non-linear fit for the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_chisq_red
+    "@id": "#Object.z_psfModel_TwoGaussian_chisq_red"
+    datatype: double
+    description: Reduced chi-squared of the best-fit model parameters (chi divided the number of data points) for the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_unknown_flag
+    "@id": "#Object.z_psfModel_TwoGaussian_unknown_flag"
+    datatype: boolean
+    description: Flag set for failures with an unexpected or unknown cause for the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_no_inputs_flag
+    "@id": "#Object.z_psfModel_TwoGaussian_no_inputs_flag"
+    datatype: boolean
+    description: Flag set for objects not fit because there were no coadd PSF inputs for the two-Gaussian PSF model (z-band).
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_is_parent_flag
+    "@id": "#Object.z_psfModel_TwoGaussian_is_parent_flag"
+    datatype: boolean
+    description: Flag set for objects not fit because they are parents for the two-Gaussian PSF model (z-band).
```

Comment:
If this is derivable from some other object table flag, I don't think we need to replicate it here, as long as we document that the PSF model Gaussian approximations and galaxy model fits don't run on parent objects.